### PR TITLE
websocket optimization

### PR DESCRIPTION
**Pull request by slackve**

### CODE REVIEW

File: sdk/subscribe_block.go [hunk 1]
```diff
@@ -66,6 +69,7 @@ func (c *SubscribeService) UnconfirmedAdded(add string) (*SubscribeTransaction,
 // address was in unconfirmed state but not anymore.
 // The message contains the transaction hash.
 func (c *SubscribeService) UnconfirmedRemoved(add string) (*SubscribeHash, error) {
```
Comment (Wondertan) [inline sdk/subscribe_block.go]: Add please Address struct as param to subscribe methods

File: sdk/websocket.go [hunk 2]
```diff
@@ -38,7 +39,8 @@ type serviceWs struct {
 type ClientWebsocket struct {
 	client    *websocket.Conn
```
Comment (Wondertan) [inline sdk/websocket.go]: I don't think this connection should be divided from the cons in the map

File: sdk/subscribe_block.go (hunk 1 above)
Comment (slackve) [reply sdk/subscribe_block.go]: With what intention! I only use the address of the account?

File: sdk/subscribe_block.go [hunk 3]
```diff
@@ -24,6 +25,90 @@ const (
...
+		} else {
+			delete(confirmedAddedChannels, s.getAdd())
+		}
+		close(chType)
+	}
+}
+
+// Unsubscribe terminates the specified subscription.
+// It does not have any specific param.
+func (c *subscribe) unsubscribe() error {
```
Comment (Wondertan) [inline sdk/subscribe_block.go]: This method panics:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1454c17]
goroutine 28 [running]:
golang.org/x/net/websocket.Codec.Send(0x16dbbb8, 0x16dbbc0, 0x0, 0x1538ca0, 0xc0003570e0, 0x0, 0x0)
	/go/src/golang.org/x/net/websocket/websocket.go:313 +0xa7
github.com/proximax-storage/proximax-nem2-sdk-go/sdk.(*subscribe).unsubscribe(0xc00028cd80, 0x0, 0xc0002aeb38)
	/go/src/github.com/proximax-storage/proximax-nem2-sdk-go/sdk/subscribe_block.go:71 +0xc0
github.com/proximax-storage/proximax-nem2-sdk-go/sdk.(*SubscribeStatus).Unsubscribe(0xc0003260e0, 0xc00021fac0, 0x40)
	/go/src/github.com/proximax-storage/proximax-nem2-sdk-go/sdk/websocket.go:349 +0x2e
github.com/proximax-storage/go-xpx/blockchain.(*BlockChain).AnnounceFromAccount.func1(0xc0001bc550, 0xc0002719e0, 0xc0002aeae0, 0xc00027ebd0, 0x1744780, 0xc0000aa008)
	/go/src/github.com/proximax-storage/go-xpx/blockchain/blockchain.go:241 +0x2aa
created by github.com/proximax-storage/go-xpx/blockchain.(*BlockChain).AnnounceFromAccount
	/go/src/github.com/proximax-storage/go-xpx/blockchain/blockchain.go:210 +0x179

Comment (slackve) [reply sdk/subscribe_block.go]: fixed up!

**Pull request merged**